### PR TITLE
npm-install: add page

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

https://github.com/tldr-pages/tldr/issues/14000

### CODE REVIEW

**Comment by Managor** [comment]
@Bucke200 I see you had more lines in the pull request that got closed. Any commands you might want to add?

File: pages/common/npm-install.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://docs.npmjs.com/cli/commands/npm-install>.
+
+- Install dependencies listed in package.json:
+
+`npm install`
+
+- Download a specific version of a package and add it to the list of dependencies in `package.json`:
+
+`npm install {{package_name}}@{{version}}`
+
```
Comment (Bucke200) [inline pages/common/npm-install.md:13]: - Install a package and update package.json:

`npm install {{package_name}} --save`

Comment (Bucke200) [reply pages/common/npm-install.md:13]: add this command

Comment (Managor) [reply pages/common/npm-install.md:13]: That seems to be the default behavior and is not needed after npm 5.

**Review by renie — APPROVED** [review]
Neat! :rocket:

**Pull request merged**